tests/base.py

```diff
@@ -375,5 +375,152 @@ def setUp(self):
       }]
     }
 
-    self.optimizely = optimizely.Optimizely(json.dumps(self.config_dict))
+    self.config_dict_with_multiple_experiments = {
```

Comment:
Shouldn't this be passed in for that one unit test that uses it?

Reply:
I think it is ok